code/modules/mob/living/carbon/xenomorph/powers.dm

```diff
@@ -893,14 +893,14 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 		M.acid_spray_act(src)
 
 
-/mob/living/carbon/Xenomorph/proc/larva_injection(mob/living/carbon/C)
-	if(!C?.can_sting())
+/mob/living/carbon/Xenomorph/proc/larva_injection(mob/living/carbon/C, precheck = TRUE)
```

Comment:
Will `precheck` ever be `TRUE`?

Reply:
As of now nope, but the defiler sting runs C.can_sting(), so I didn't feel like double-checking it. But it's still around, being part of different proc.

Reply:
Any reason for the extra var and check then?